src/Tools/BuildBoss/Program.cs

```diff
@@ -138,14 +138,14 @@ private static bool ProcessSolution(string solutionFilePath, bool isPrimarySolut
 
         private static bool ProcessTargets(string repositoryDirectory)
         {
-            var targetsDirectory = Path.Combine(repositoryDirectory, @"eng\targets");
+            var targetsDirectory = Path.Combine(repositoryDirectory, Path.Combine("eng", "targets"));
             var checker = new TargetsCheckerUtil(targetsDirectory);
             return CheckCore(checker, $"Targets {targetsDirectory}");
         }
 
         private static bool ProcessStructuredLog(string artifactsDirectory, string configuration)
         {
-            var logFilePath = Path.Combine(artifactsDirectory, $@"log\{configuration}\Build.binlog");
+            var logFilePath = Path.Combine(Path.Combine(Path.Combine(artifactsDirectory, "log"), "{configuration}"), "Build.binlog");
```

Comment:
It looks like `"{configuration}"` was meant to be `$"{configuration}"`.

Reply:
yes

Reply:
I think the issue wasn't resolved here--we don't want to delete parameter `configuration`. What we want is to actually use it to create the log file path. Which will happen if we change `"{configuration}"` to `$"{configuration}"`.

Reply:
I now changed the code to use the source string, I think It's the best form.